pkg/server/endpoints/node/handler_test.go

```diff
@@ -314,6 +315,36 @@ func (s *HandlerSuite) TestAttestSuccess() {
 	s.Empty(s.getNodeSelectors(agentID))
 }
 
+func (s *HandlerSuite) TestAttestAgentless() {
+	//  Create a registration entry to return with the SVID update
+	s.createRegistrationEntry(&common.RegistrationEntry{
+		ParentId: agentlessID,
+		SpiffeId: workloadID,
+	})
```

Comment:
I'm not sure I understand why we need to create a registration entry for this? It should work without one, no?

Reply:
yes, I forgot to remove it 🤦‍♂️